tools/check-abi.sh

```diff
@@ -3,7 +3,7 @@
 set -eu
 
 default_base_version="$(git describe --match "v*.*.*" --abbrev=0)"
-default_new_version="master"
+default_new_version="HEAD"
```

Comment:
```suggestion
default_new_version="$(git rev-parse HEAD)"
```

Reply:
I'm not entirely sure what you meant here, but I believe I fixed it